draft-ietf-cose-x509.md

```diff
@@ -29,6 +29,7 @@ informative:
   RFC8610:
   RFC3986:
   I-D.ietf-anima-constrained-voucher:
+  RFC6838:
```

Comment:
It is not entirely intuitive to me that this reference should be informative, but I trust your judgement on that one.

Reply:
I agree that informative is okay here.